### ISSUE TITLE
(Collaborating on) updating to NeoForge 1.21.1

### ISSUE THREAD

**Issue by HoldYourWaffle** [Respace-io/pvp-flagging#2]
This mod looks like it'd be a great fit for our server, thanks for sharing your hard work!

Are you planning to update it to NeoForge 1.21.1?
My last modding experience dates back to the 1.12 era, but I'd love to collaborate on this if you're up for it.

**Comment by lab3** [comment]
Yes, I do want to port it I just haven't had the time to dig into it yet.

**Comment by lab3** [comment]
@HoldYourWaffle I just pushed a 1.21 branch with the port. I haven't done any testing yet.  Could use the help if you are up for it

**Comment by lab3** [comment]
You can build the branch or grab the snapshot here https://code.redspace.io/#/snapshots/io/redspace/pvp_flagging/1.21.1-1.0-SNAPSHOT
